### PR TITLE
Adds badge and spider diagram SVG to the Women's Euros 2022 competition

### PR DESCRIPTION
**Pull request by AshCorr**
Reverts guardian/frontend#25100 and adds a badge for the womens euros 2022 front

# Screenshots
| image |
|:-:|
| *Womens Euros 2022 badge in an Article* |

| image |
|:-:|
| *SVG in the Womens Euros 2022 spider diagram* |

### CODE REVIEW

**Review by HarryFischer — APPROVED** [review]

**Review by oliverlloyd — APPROVED** [review]
I have looked though these changes and don't see anything of concern to my non Scala expert eyes. I'm happy to sign this off if Harry is happy and things are testing well 👍

File: sport/app/football/views/fragments/teamForm.scala.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 @competition.teamResults(teamId).takeRight(5).map { result =>
     @if(result.won) {
         <span
+            tabindex="0"
```
Comment (jamesgorrie) [inline sport/app/football/views/fragments/teamForm.scala.html:9]: This shouldn't be here.

**Review by jamesgorrie — APPROVED** [review]
Just a tabindex to remove

**Comment by mxdvl** [comment]
That badge's SVG could be made a tad more semantic, but it's nice to see it using the multiply blend mode!

**Pull request merged**